docs/data/material/components/autocomplete/autocomplete.md

```diff
@@ -225,11 +225,13 @@ Before you can start using the Google Maps JavaScript API and Places API, you ne
 This demo has limited quotas to make API requests. When your quota exceeds, you will see the response for "Paris".
 :::
 
-## Custom Single Value Rendering
+## Single value rendering
 
-By default, when `multiple={false}`, the selected option is displayed as plain text inside the input. The `renderValue` prop allows you to customize how the selected value is rendered. This can be useful for adding custom styles, displaying additional information, or formatting the value differently.
+By default (when `multiple={false}`), the selected option is displayed as plain text inside the input.
+The `renderValue` prop allows you to customize how the selected value is rendered.
+This can be useful for adding custom styles, displaying additional information, or formatting the value differently.
 
-- The `getItemProps` callback provides props like `data-item-index`, `disabled`, `tabIndex` and others. These props should be spread onto the rendered component for proper accessibility.
+- The `getItemProps` getter provides props like `data-item-index`, `disabled`, `tabIndex` and others. These props should be spread onto the rendered component for proper accessibility.
```

Comment:
Ok, https://github.com/downshift-js/downshift also calls this a getter, let's move with this.